doc/development/html_themes/index.rst

```diff
@@ -221,6 +221,62 @@ If your theme package contains two or more themes, please call
    ``sphinx.html_themes`` entry_points feature.
 
 
+Styling with CSS
+----------------
+The **stylesheets** setting allows to add custom CSS files to a theme.
```

Comment:
```suggestion
The :confval:`stylesheets` setting can be used to add custom CSS files to a theme.
```

Reply:
My mistake, sorry - it seems that this isn't a traditional config setting; it's a theme configuration setting, and those appear to be different.  I'll try to figure out whether it _could_ be made reference-able.. but perhaps that should be reverted.